website/content/docs/configuration/session-recording/configure-worker-storage.mdx

```diff
@@ -64,6 +64,51 @@ worker {
 }
 ```
 
+## Remote storage state
+
+Workers that are configured for session recording storage must have access to the configured storage bucket. Storage bucket represents a bucket in an external storage provider. The remote storage state of a worker is determined by the availability of the storage bucket.
+Workers are configured to report their permission state for actions using the storage bucket configured.
+
+The permission states determine actions that the worker can perform on the storage bucket. The permission states are:
+- `read`: The worker can read from the storage bucket. Session recording playback is possible if the state is healthy.
+- `write`: The worker can write to the storage bucket. Session recording is possible if the state is healthy.
+- `delete`: The worker can delete from the storage bucket. Session recording cleanup is possible if the state is healthy.
```

Comment:
```suggestion
Workers can have the following permission states:

-`read`: The worker can read from the external storage. It can also play back recorded sessions, if the remote storage state is `available`.
- `write`: The worker can write to the external storage. It can also record sessions, if the remote storage state is `available`.
- `delete`: The worker can delete data from the external storage. It can also clean up recorded sessions, if the remote storage state is `available`.
```